examples/rust/timezonets/src/lib.rs

```diff
@@ -0,0 +1,745 @@
+use chrono::{DateTime, Datelike, Duration, Months, NaiveDateTime, Timelike, Utc};
+use chrono_tz::Tz;
+use itertools::Itertools;
+
+wit_bindgen_rust::export!("tsz.wit");
+
+struct Tsz;
+
+fn contains_tz(fmt: &str) -> bool {
+    if fmt.contains("%z")
```

Comment:
you could reduce this to to just the conditions (without the if...else form) since rust will just return the value last line of the function.